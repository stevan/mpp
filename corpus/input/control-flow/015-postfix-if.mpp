$x = 5 if $condition;