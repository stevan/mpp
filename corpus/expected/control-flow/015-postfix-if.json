[
  {
    "type": "If",
    "condition": {
      "type": "Variable",
      "name": "$condition"
    },
    "thenBlock": [
      {
        "type": "Assignment",
        "left": {
          "type": "Variable",
          "name": "$x"
        },
        "operator": "=",
        "right": {
          "type": "Number",
          "value": "5"
        }
      }
    ],
    "elseIfClauses": []
  }
]